### PR TITLE
List layers in projects view

### PR DESCRIPTION
**Pull request by propensive**

### CODE REVIEW

**Comment by odisseus** [comment]
That's quite a lot of changes. Are all of them actually related to the issue #1396?

**Comment by propensive** [comment]
Indirectly, but the original issue, as described in #1396, turned out not to be the solution I wanted. What I'm doing right now is more experimental.

Apart from the layer issue, does it conflict with anything you're doing?

**Comment by odisseus** [comment]
At the moment I'm not doing anything complicated, so I don't think there will be significant conflicts.

However, I think that some of the changes in this pull request (at least the renaming of arguments) can be extracted into separate pull requests.

File: test/passing/repo-change-branch/script [hunk 1]
```diff
@@ -0,0 +1,6 @@
+fury clean repositories
```
Comment (odisseus) [inline test/passing/repo-change-branch/script:1]: The tests that use a pre-defined layer don't reduce complexity. They just hide it away in a format that's difficult to inspect. Additionally, these layers may become invalid as the layer format evolves.

File: test/passing/repo-change-branch/description [hunk 2]
```diff
@@ -0,0 +1 @@
+Change to a different branch of a repository
```
Comment (odisseus) [inline test/passing/repo-change-branch/description:1]: This test is missing a pre-defined layer.

File: test/passing/repo-branch-not-found/description [hunk 3]
```diff
@@ -0,0 +1 @@
+Attempt to checkout a branch which does not exist.
```
Comment (odisseus) [inline test/passing/repo-branch-not-found/description:1]: Looks like this test is missing a pre-defined layer.

File: test/passing/repo-change-tag/description [hunk 4]
```diff
@@ -0,0 +1 @@
+Change to a different tag of a repository
```
Comment (odisseus) [inline test/passing/repo-change-tag/description:1]: This test is missing a pre-defined layer.

**Pull request merged**